redshift_sqlalchemy/dialect.py

```diff
@@ -189,17 +279,106 @@ def set_isolation_level(self, connection, level):
 
         connection.set_isolation_level(level)
 
-    def _get_column_info(self, *args, **kwargs):
+    def _get_column_info(self, col, domains):
         column_info = super(RedshiftDialect, self)._get_column_info(
-            *args,
-            **kwargs
-        )
+            col.name, col.format_type, col.default, col.notnull,
+            domains, [], col.schema)
         if isinstance(column_info['type'], VARCHAR):
             if column_info['type'].length is None:
                 column_info['type'] = NullType()
-
+        if 'info' not in column_info:
+            column_info['info'] = {}
+        if col.encoding and col.encoding != 'none':
+            column_info['info']['encode'] = col.encoding
```

Comment:
you can do these four lines in two using setdefault:
```
if col.encoding and col.encoding != 'none':
    column_info.setdefault('info', {})['encode'] = col.encoding
```

though it doesn't always improve readability, so leaving as is, is fine too.